themes/default/content/docs/concepts/vs/terraform/_index.md

```diff
@@ -224,6 +225,10 @@ Aliases help facilitate refactoring by allowing you to modify certain properties
 
 Transformations, which are unique to Pulumi, allow you to programmatically set or override the input properties of resources belonging to a particular collection, such as the child resources of a Pulumi component or even all of the resources belonging to a stack. Transformations make it easy to apply consistent settings across your infrastructure without having to manipulate the properties of individual resources. To learn more, see [Transformations](/docs/concepts/options/transformations/) in the Resource documentation.
 
+### Terminology and Commands
+
+If you’re familiar with Terraform, learning Pulumi terminology and commands is simple. Many of the existing Terraform vocabulary and commands that you already know have direct equivalents in Pulumi. To learn more, see [Pulumi terms & command equivalents](/docs/concepts/vs/terraform/terminology/) in the Resource documentation to find a list of common Terraform terms and CLI commands along with their Pulumi equivalents.
```

Comment:
Minor nit, but since these don't live in the resource documentation (which is more what registry API docs are):

```suggestion
If you’re familiar with Terraform, learning Pulumi terminology and commands is simple. Many of the existing Terraform vocabulary and commands that you already know have direct equivalents in Pulumi. To learn more, see [Pulumi terms & command equivalents](/docs/concepts/vs/terraform/terminology/) for a list of common Terraform terms and CLI commands along with their Pulumi equivalents.
```